Common.php

```diff
@@ -2,13 +2,6 @@
 function response($input)
 {
     #error_reporting(0);
-    if (is_array($input[0])) {
-        foreach ($input as $array) {
-            $resp = ['data' => $array['data'], 'message' => $array['message'], 'status' => $array['status']];
-            echo json_encode($resp);
-        }
-        exit;
-    }
     $resp = ['data' => $input['data'], 'message' => $input['message'], 'status' => $input['status']];
```

Comment:
Your input array is totally the same as $resp variable.
You can do just echo json_encode($input);
Or to be sure that you send only this fields do array_intersect for needed keys.